Library/Homebrew/cleaner.rb

```diff
@@ -37,7 +37,7 @@ def observe_file_removal(path)
   end
 
   # Removes any empty directories in the formula's prefix subtree
-  # Keeps any empty directions projected by skip_clean
+  # Keeps any empty directions protected by skip_clean
```

Comment:
directions -> directories?

Reply:
haha, oh wow, that sentence really went off the rails :D